cpp/ql/src/Likely Bugs/UnderspecifiedFunctions/TooManyArguments.ql

```diff
@@ -0,0 +1,29 @@
+/**
+ * @name Call to function with extraneous parameters
+ * @description A function call to a function passed more arguments than there are
+ *              declared parameters of the function.  This may indicate
+ *              that the code does not follow the author's intent.
+ * @kind problem
+ * @problem.severity warning
+ * @precision very-high
+ * @id cpp/too-many-arguments
```

Comment:
The new query id (`cpp/too-many-arguments`) is better than the old one (`cpp/futile-params`) and consistent with the others, but that's not a good enough reason to rename a query. Historical results from all projects on LGTM are keyed to that name, and users may have used the name to suppress individual alerts or the whole query. We're releasing 1.20 with the `cpp/futile-params` id, so this name will go to customers too. Changing a query id is like deleting a query and making a new one, so we only do it when the results have changed so much that old configurations aren't expected to make sense for the revised query.

Reply:
Ok, I'll rename it back.  Please tell me that the new file folder structure is still OK?

Reply:
Yes, it should now be fine to move a query file. We had some legacy products that depended on file names never changing, but will be unsupported in release 1.21 and later.

There are still some references to `FutileParams.ql` that you'll have to change. It's listed in `cpp/config/suites/c/correctness` and `cpp/config/suites/cpp/correctness`.

Is there a strong reason not to put a space in the `UnderspecifiedFunctions` directory name? As much as I dislike spaces in file names personally, we do name these directories with spaces by convention.

Reply:
I'll fix up the correctness files.  As @geoffw0 pointed out, there is also a changelog that needs to be updated.

As far as spaces, there is no strong reason (apart from not liking it) - it seems that the <code>src</code> and <code>test</code> trees contain both spaced and non-spaced names.  I'll change them if that's the convention to follow.